src/Workspaces/CoreTest/app.config

```diff
@@ -13,36 +13,4 @@
       </listeners>
     </trace>
   </system.diagnostics>
-  <runtime>
```

Comment:
Can't.  We're still generating these redirects into it at build time and if we don't have the file, it doesn't know to copy the defaults.